engines/tika/src/main/java/org/alfresco/transform/tika/transformers/ArchiveTransformer.java

```diff
@@ -24,25 +24,17 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  * #L%
  */
-package org.alfresco.transformer;
+package org.alfresco.transform.tika.transformers;
 
-import org.springframework.boot.test.context.SpringBootTest;
+import org.apache.tika.parser.Parser;
+import org.springframework.stereotype.Component;
 
-/**
- * Tests ImageMagickController with a server test harness.
- */
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class ImageMagickHttpRequestTest extends AbstractHttpRequestTest
+@Component
+public class ArchiveTransformer extends AbstractTikaTransformer
 {
     @Override
-    protected String getTransformerName()
-    {
-        return "ImageMagick";
-    }
-
-    @Override
-    protected String getSourceExtension()
+    protected Parser getParser()
     {
-        return "jpg";
+        return tika.packageParser;
```

Comment:
`packageParser` is a static field and should be accessed in a static way.

```suggestion
        return Tika.packageParser;
```